.gitattributes

```diff
@@ -1,8 +1,10 @@
+* text=auto eol=lf
+
 # Source files should not be normalized to avoid confusing clang-format-diff.py
-*.c text eol=lf
-*.cc text eol=lf
-*.cpp text eol=lf
-*.h text eol=lf
+*.c -diff
```

Comment:
According to the documentation this would "[...] cause Git to generate Binary files differ (or a binary patch, if binary patches are enabled) instead of a regular diff."

That's not what we want.
We just want git not to change the line endings.